packages/geo/src/lib/catalog/catalog-browser/catalog-browser-layer.component.ts

```diff
@@ -22,7 +27,7 @@ export class CatalogBrowserLayerComponent {
   /**
    * Whether the layer is already added to the map
    */
-  @Input() added: boolean;
+  @Input() added = false;
```

Comment:
I like to type inputs even when they have a default value for compodoc.

Reply:
done